cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -290,6 +290,17 @@ jobs:
 
 {{%- endif %}}
 
+{{%- for job in user_publish_jobs %}}
+
+  user-publish-job-{{{ job|safe }}}:
```

Comment:
The full `user-publish-job-` prefix seems a bit cumbersome, `publish-{slug}` too dangerous for your tastes? Or maybe just `custom-{slug}` since the user will likely have names like `publish-npm.yml`?

Reply:
`custom-` sounds good to me.